photon-ml/src/integTest/scala/com/linkedin/photon/ml/sampler/DefaultDownSamplerIntegTest.scala

```diff
@@ -87,9 +83,11 @@ class DefaultDownSamplerIntegTest extends SparkTestUtils {
     } else if (downSamplingRate == 1.0) {
       Assert.assertEquals(numInstancesInSampled, numTimesToRun * numInstancesToGenerate)
     } else {
-      Assert.assertEquals(numInstancesInSampled * 1.0 / numTimesToRun / numInstancesToGenerate,
-        downSamplingRate,
-        tolerance)
+      val mean = numTimesToRun*numInstancesToGenerate*downSamplingRate
+      val variance = mean*(1-downSamplingRate)
```

Comment:
There should be spaces between the operators. 
http://docs.scala-lang.org/style/method-invocation.html